main.js

```diff
@@ -143,11 +158,17 @@ function createLabel(htmlForName, labelText){
     return label;
 } 
 
-function createInput(name, inputId = name, placeholder = name, type = name){
+function createInput(value = "", name, inputId = name, placeholder = name, type = name){
     const input = createBaseElement({tagName: "input", tagId: inputId});
+    input.value = value;
     input.type = type;
     input.name = name;
     input.placeholder = placeholder;
     return input;
 }
 
+function createParagraph(paragraphId, errorText, className = "error"){
```

Comment:
Why `createParagraph()` has default class 'error'? And why it has `errorText` argument?

Is it common `createParagraph()` or `createErrorParagraph()`? I just can't understand 🚑

Reply:
Done